### ISSUE TITLE
How to let a sub graph don't update?

### ISSUE THREAD

**Issue by yang182** [clab/dynet#1588]
My loss function is a KL distance. loss = pi * log(pi) - pi * log(qi). If I sometimes want update subgraph from qi only. Note the pi is another subgraph. What should I do?   The problem come from semi-supervised learning  article "Semi-SupervisedSequenceModelingwithCross-ViewTraining
"

**Comment by FilippoC** [comment]
you can use the nobackprop operation : https://dynet.readthedocs.io/en/latest/operations.html#_CPPv4N5dynet10nobackpropERK10Expression

**Comment by yang182** [comment]
> you can use the nobackprop operation : Thanks

**Comment by yang182** [comment]
@FilippoC   Thanks
